ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js

```diff
@@ -0,0 +1,9040 @@
+              _.v)(a.Vg, a)) && Ze(a, new hb.bb(hb.bb.g.Ce)),
+              Ue(a))
+          }
+            , na = function(a) {
+              if (a.o) {
+                  var b = a.o.mb()
+                    , c = [];
+                  if (a.s[b]) {
+                      for (var d = _.Sa(Object.keys(a.s[b])), e = d.next(); !e.done; e = d.next()) {
+                          e = e.value;
+                          var f = a.g[e];
+                          f && !f.g && (a.Ih(b, e),
+                          c.push(e))
+                      }
+                      Xe(a, c)
+                  }
+                  a.zb() || (a.g[b].onLoad((0,
+                  _.v)(a.Vg, a)) && Ze(a, new hb.bb(hb.bb.g.Ce)),
+                  _.va(a.C, b),
+                  _.va(a.j, b),
+                  0 === a.j.length && bf(a),
+                  a.W && b == a.W && (a.K.g || a.K.callback()),
+                  Ue(a),
+                  a.o = null)
+              }
+          }
+            , We = function(a, b) {
+              if (_.ta(a.j, b))
+                  return !0;
+              for (var c = 0; c < a.h.length; c++)
+                  if (_.ta(a.h[c], b))
+                      return !0;
+              return !1
+          };
+          Qe.prototype.load = function(a, b) {
+              return Xe(this, [a], b)[a]
+          }
+          ;
+          var la = function(a) {
+              var b = _.fa;
+              b.o && "synthetic_module_overhead" === b.o.mb() && (na(b),
+              delete b.g.synthetic_module_overhead);
+              b.g[a] && df(b, b.g[a].Dc() || [], function(c) {
+                  c.g = new Ib;
+                  _.va(b.j, c.mb())
+              }, function(c) {
+                  return !c.g
+              });
+              b.o = b.g[a]
+          };
+          Qe.prototype.fa = function() {
+              $e(this, new hb.bb(hb.bb.g.TIMEOUT));
+              bf(this)
+          }
+          ;
+          var $e = function(a, b) {
+              1 < a.l.length ? a.h = a.l.map(function(c) {
+                  return [c]
+              }).concat(a.h) : Ze(a, b)
+          }
+            , Ze = function(a, b) {
+              var c = a.l;
+              a.j.length = 0;
+              for (var d = [], e = 0; e < a.h.length; e++) {
+                  var f = a.h[e].filter(function(l) {
+                      var m = cf(this, l);
+                      return _.Sb(c, function(n) {
+                          return _.ta(m, n)
+                      })
+                  }, a);
+                  xa(d, f)
+              }
+              for (e = 0; e < c.length; e++)
+                  _.ua(d, c[e]);
+              for (e = 0; e < d.length; e++) {
+                  for (f = 0; f < a.h.length; f++)
+                      _.va(a.h[f], d[e]);
+                  _.va(a.C, d[e])
+              }
+              var g = a.ma.error;
+              if (g)
+                  for (e = 0; e < g.length; e++) {
+                 
```

Comment:
A lot of `googleapis.com` endpoints here. Won't brave us it to send requests to this domain?